### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by misazr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,18 +10,50 @@ A ruby REST client that lets you feel like in heaven when consuming APIs.
...
+3. Choose which HTTP adapter you want to use
+
+        gem 'excon'
 
 ## Usage
 
 ### HTTP Client Library
 
-There is no dependency on a specific HTTP client library but the client has been tested with [Faraday](https://github.com/lostisland/faraday) only. You can use any other client library as long as it has the same API as Faraday.
+There is no dependency on a specific HTTP client library but the client has been tested with combination of [Faraday](https://github.com/lostisland/faraday) and [Excon](https://github.com/excon/excon) only. You can use any other client library as long as it has the same API as Faraday.
```
Comment (luxflux) [inline README.md]: IMHO this isn't totally correct. The only dependency is faraday, what kind of adapter is being used in Faraday doesn't matter for RiP. Basically RiP supports every adapter that Faraday supports.

File: README.md [hunk 2]
```diff
@@ -10,18 +10,50 @@ A ruby REST client that lets you feel like in heaven when consuming APIs.
...
+```ruby
+class Resource
+  include RESTinPeace
+    
+  rest_in_peace do
+    use_api ->() do
+      ::Faraday.new(url: 'http://rip.dev', headers: { 'Accept' => 'application/json' }) do |faraday|
+        faraday.request :json
+        faraday.response :json
+        faraday.adapter :excon # make requests with Net::HTTP
```
Comment (luxflux) [inline README.md]: I think the comment should be `# make requests with Excon`.

File: README.md [hunk 3]
```diff
@@ -40,7 +72,7 @@ end
 
 #### API Endpoints
 
-You need to define all the API endpoints you want to consume with `RESTinPeace`. Currently the four HTTP verbs `GET`, `POST`, `PATCH` and `DELETE` are supported.
+You need to define all the API endpoints you want to consume with `REST-in-Peace`. Currently the four HTTP verbs `GET`, `POST`, `PATCH` and `DELETE` are supported.
```
Comment (luxflux) [inline README.md]: I think `PUT` is supported as well, maybe add this here as well...

File: README.md [hunk 4]
```diff
@@ -220,6 +232,70 @@ module MyClient
 end
 ```
 
+```ruby
```
Comment (luxflux) [inline README.md]: Maybe add a title here to explain what kind of an example this is.

File: README.md [hunk 5]
```diff
@@ -10,18 +10,51 @@ A ruby REST client that lets you feel like in heaven when consuming APIs.
...
+3. Choose which HTTP adapter you want to use
+
+        gem 'excon'
 
 ## Usage
 
 ### HTTP Client Library
 
-There is no dependency on a specific HTTP client library but the client has been tested with [Faraday](https://github.com/lostisland/faraday) only. You can use any other client library as long as it has the same API as Faraday.
+This gem dependents on Faraday - HTTP client library. REST-in-Peace has been tested with combination
```
Comment (neckhair) [inline README.md]:
```markdown
This gem depends on the HTTP client library [Faraday](https://github.com/lostisland/faraday).
```

```
in combination with
```

File: README.md [hunk 6]
```diff
@@ -40,7 +73,7 @@ end
 
 #### API Endpoints
 
-You need to define all the API endpoints you want to consume with `RESTinPeace`. Currently the four HTTP verbs `GET`, `POST`, `PATCH` and `DELETE` are supported.
+You need to define all the API endpoints you want to consume with `REST-in-Peace`. Currently the four HTTP verbs `GET`, `POST`, `PUT`, `PATCH` and `DELETE` are supported.
```
Comment (neckhair) [inline README.md]: The five verbs

File: README.md [hunk 7]
```diff
@@ -220,6 +235,72 @@ module MyClient
 end
 ```
 
+#### Configured Fabric class using the Faraday
```
Comment (neckhair) [inline README.md]: using Faraday

**Pull request merged**